idd/Energy+.idd.in

```diff
@@ -24996,10 +24996,12 @@ AirflowNetwork:Distribution:Component:OutdoorAirFlow,
       \min-fields 3
       \memo This object includes the outdoor air flow rate set by the Controller:OutdoorAir
       \memo object in the airflow network.
-      \unique-object
  A1 , \field Name
       \required-field
       \reference AFNOutdoorAirFlowNames
+ A2 , \field Outdoor Air Mixer Name
+      \required-field
+      \object-list OutdoorAirMixers
```

Comment:
Transition is needed, right?

Reply:
@Myoldmopar Yes. I am going to make transition rules in Rules8-9-0-to-9-0-0.md